docs/src/file_types/pmd.md

```diff
@@ -0,0 +1,125 @@
+# PMD
+
+> 💡 **Tip:** We have a syntax highlighter for .pmd files available on [Visual Studio's Marktetplace](https://marketplace.visualstudio.com/items?itemName=pedromxavier.psr-pmd)
+
+> ⚠ **Warning:** The syntax for PMD files is case-sensitive 
```

Comment:
Documenter.jl has a few built-in [admonition options](https://documenter.juliadocs.org/stable/showcase/#Admonitions). Let's use them!

```suggestion
!!! tip
    We have a syntax highlighter for .pmd files available on [Visual Studio's Marktetplace](https://marketplace.visualstudio.com/items?itemName=pedromxavier.psr-pmd)

!!! warning
    The syntax for PMD files is case-sensitive 
```